.travis.yml

```diff
@@ -10,6 +10,10 @@ before_script:
   - cp keys/saml.key.enc.example keys/saml.key.enc
   - bin/rake db:setup --trace
   - bin/rake assets:precompile
+after_script:
+  - npm install -g pa11y-crawl
+  - bundle exec rails s -d
+  - pa11y-crawl --standard Section508 --ci http://localhost:3000
```

Comment:
Does `rails server -d` block until the server is ready?  i.e. will the server be ready when the crawl starts?

I'm OK with this change if you can move pa11y-crawl to the dev dependencies of package.json.   I can make a general travis cleanup PR as a followup.